assets/javascripts/views/search/search.coffee

```diff
@@ -113,6 +114,10 @@ class app.views.Search extends app.View
     @externalSearch "https://stackoverflow.com/search?q="
     return
 
+  duckduckgo: =>
+    @externalSearch "https://duckduckgo.com/?q="
```

Comment:
Also, could you change this to `https://duckduckgo.com?t=devdocs&q=` to help DuckDuckGo [see where their searches are coming from](https://duck.co/help/privacy/t)?